Gemfile

```diff
@@ -5,3 +5,4 @@ gemspec
 
 gem "rake", "~> 12.0"
 gem "rspec", "~> 3.0"
+gem "rubocop", "~> 0.90"
```

Comment:
Otherwise running `bundle install` to update the rubocop-fairmoney spec will update Rubocop to 1.13

Reply:
Can't we use at least rubocop 0.93? (monolith version)

Reply:
If we upgrade Rubocop here I would rather go to 1.13 and make sure it works on all repos